Sources/TuistKit/Services/FetchService.swift

```diff
@@ -61,6 +69,36 @@ final class FetchService {
         logger.info("Plugins resolved and fetched successfully.", metadata: .success)
     }
     
+    private func updateDependencies(path: AbsolutePath) throws {
```

Comment:
The code is almost identical to the fetchDependencies, should we try to avoid repeating it?

Reply:
oh yeah, absolutely!